README.md

```diff
@@ -1,2 +1,167 @@
-# IDR_stream
-Software for feature extraction from IDR image data
+# idrstream
+
+Software for feature extraction from IDR image data!
```

Comment:
Can you be a bit more specific?

i.e. - comment on what it means to be a "stream"

Reply:
Done in https://github.com/WayScience/IDR_stream/pull/2/commits/8b96b3928f1f755707410a5066f184685599579f